src/components/feed/CreateFeed/BasicInformation.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+
+import { Form, FormGroup, TextInput, TextArea } from '@patternfly/react-core';
+import Client, { Tag } from '@fnndsc/chrisapi';
+import { Typeahead } from 'react-bootstrap-typeahead';
+
+export declare var process: { 
```

Comment:
Is this needed here? Is process defined in globals?

Reply:
process is defined in globals, but `process.env.REACT_APP_CHRIS_UI_URL` is typed as `string | undefined` without this declaration. Since Client takes a `string` as the first param, it needs to be typed as a `string`. An alternative could be something like `new Client(process.env.REACT_APP_CHRIS_UI_URL || '', { ... })`. Would that be better?